src/marshmallow/experimental/meta.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import typing
+
+if typing.TYPE_CHECKING:
+    from marshmallow.fields import Field
+    from marshmallow.schema import SchemaMeta
+    from marshmallow.types import UnknownOption
+
+
+@typing.overload
+def meta(
+    *bases: SchemaMeta,
```

Comment:
Should this be 

```suggestion
    *bases: Schema.Meta,
```

? Surprised mypy doesn't raise an error in the tests tho 🤔